gatsby-site/src/components/Contact/contact.css

```diff
@@ -0,0 +1,85 @@
+:root {
+    --colarkfontcolor: #0111FC;
+    --colarkfont: 'Mukta Malar', sans-serif;
+    --background: #FFFFFF;
+}
+
+.form-container {
+    width: 100%;
+    text-align: center;
+    padding: 40px;
+}
+
+.left-image {
+    width: 50%;
+    margin: 20px;
+}
+
+.right-form {
+    width: 40%;
+    word-break: keep-all;
+}
+
+.right-form, .left-image{
+    display: inline-block;
+    padding: 50px;
+    align-self: center;
+}
+
+.submitbtn {
+    float: right;
+    width: 40%;
+    padding: 0;
```

Comment:
Is changing the padding like this a good idea? It seems like it is off-center vertically so maybe there is a better way to make it the size you want.